<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuboInteractBalanceViewer</class>
 <widget class="QDialog" name="HuboInteractBalanceViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="glLayout"/>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>480</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
